app/controllers/patients_controller.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+# Define the patients controller class.
+class PatientsController < ApplicationController
+  include AccessHelper
+
+  before_action :confirm_logged_in
+
+  # GET /patients
+  # Display patients list.
+  def index
+    @patients = Patient.sorted_by_id
+  end
+
+  # GET /patients/:id
+  # Display patient information.
+  def show
+    @patient = Patient.find(params[:id])
+  end
+
+  # GET /patients/new
+  # Display patient creation form with default email.
+  def new
+    @patient = Patient.new(email: 'default@email.com')
```

Comment:
default values for models can be added to the migrations

Reply:
Yup, this was just for practicing that a creation form can be pre-populated with something.